init.lua

```diff
@@ -115,13 +115,71 @@ minetest.register_node("crafting_bench:workbench",{
 	end,
 	on_blast = function(pos) end,
 	on_metadata_inventory_move = function(pos, from_list, from_index, to_list, to_index, count, player)
-		minetest.log("action", player:get_player_name().." moves stuff in workbench at "..minetest.pos_to_string(pos))
+		local meta = minetest.get_meta(pos)
+		local inv = meta:get_inventory()
+		local stack = inv:get_stack(to_list, to_index)
+		stack:set_count(count)
+		minetest.log("action", player:get_player_name().." moves "..stack:to_string().." in workbench at "..minetest.pos_to_string(pos))
```

Comment:
This looks very hacky. What's the difference to what Minetest currently does?

Reply:
I'll remove this callback since there's no advantage by having it.